Sources/Views/ReloadableViewLayoutAdapter.swift

```diff
@@ -127,19 +145,26 @@ open class ReloadableViewLayoutAdapter: NSObject, ReloadableViewUpdateManagerDel
                 let header = sectionLayout.header.map(layoutFunc)
                 let footer = sectionLayout.footer.map(layoutFunc)
                 var items = [LayoutArrangement]()
+                var insertedIndexPaths = [IndexPath]()
 
                 for (itemIndex, itemLayout) in sectionLayout.items.enumerated() {
                     if operation?.isCancelled ?? true {
                         return
                     }
 
                     items.append(layoutFunc(itemLayout))
+                    insertedIndexPaths.append(IndexPath(item: itemIndex, section: sectionIndex))
 
-                    let partialSection = Section(header: header, items: items, footer: footer)
-                    var partialArrangement = pendingArrangement
-                    partialArrangement.append(partialSection)
-                    let insertedIndexPath = IndexPath(item: itemIndex, section: sectionIndex)
-                    updateManager.apply(partialArrangement: partialArrangement, insertedIndexPath: insertedIndexPath)
+                    if (itemIndex <= ReloadableViewLayoutAdapter.incrementalUpdateChunkingThreshold
```

Comment:
Can you add a comment that explains why it updates for each item for the first chunk and then chunks at a time after that